app/assets/javascripts/location_search.js

```diff
@@ -11,7 +11,8 @@ window.ST = window.ST || {};
     var form = document.querySelector(selectors.form);
     var autocomplete = new window.google.maps.places.Autocomplete(searchInput, { bounds: { north: -90, east: -180, south: 90, west: 180 } });
     var locationQueryMade = false;
-    autocomplete.setTypes(['geocode']);
```

Comment:
Why is this removed? Sorry, if I haven't been clear`setTypes` is a different setting than `setFields`, both need to be set.